tchannel-crossdock/pom.xml

```diff
@@ -23,6 +23,13 @@
         <dependency>
             <groupId>com.uber.jaeger</groupId>
             <artifactId>jaeger-core</artifactId>
+            <version>${com.uber.jaeger.version}</version>
```

Comment:
remove the version and exclusions. these come from parent pom dependency management.

Reply:
done